backend/api/views.py

```diff
@@ -1,3 +1,36 @@
-from django.shortcuts import render
+from rest_framework import viewsets, mixins
+from serializers import BusinessSerializer
+from ickly import secret_settings
+from sodapy import Socrata
+
+class Business(object):
+    def __init__(self, camis, dba, cuisine_description, address):
+        self.camis = camis
+        self.dba = dba
+        self.cuisine_description = cuisine_description
+        self.address = address
+
+
+class BusinessesViewSet(mixins.ListModelMixin,
+                        viewsets.GenericViewSet):
+
+    serializer_class = BusinessSerializer
+
+    def get_queryset(self):
+        client = Socrata("data.cityofnewyork.us", secret_settings.APP_TOKEN)
```

Comment:
Hmmm. Nope, its working in my environment. 

I also don't get the warning you have below, however, that warning is related to why everything broke for me and i was getting an 'error 54, 'Connection reset by peer' before i reset my python dev environment and got my brewed python to use brewed OpenSSL instead of the outdated one. 

Does the warning go away if you install with the secure extra?: https://urllib3.readthedocs.io/en/latest/user-guide.html#certificate-verification-in-python-2